include/zephyr/bluetooth/gatt.h

```diff
@@ -2117,7 +2103,7 @@ typedef void (*bt_gatt_subscribe_func_t)(struct bt_conn *conn, uint8_t err,
 					 struct bt_gatt_subscribe_params *params);
 
 /** Subscription flags */
-enum {
+enum bt_gatt_subs_flag {
```

Comment:
Similar to how you added references like `Flags of type @ref bt_gatt_attr_write_flag` you could do the same for the subs flag